docs/self-hosted/quickstart.md

```diff
@@ -66,11 +71,27 @@ file.
 
 ## Run Directus
 
-Run the following in your terminal:
+::: tabs
+
+== macOS/Linux
+
+Open the Terminal and run the following commands one line at a time:
+
+```
+cd Desktop/directus
+docker compose up
+```
+
+== Windows
+
+Open the Command Line and run the following commands one line at a time:
 
 ```
+cd /d %userprofile%\Desktop\directus
```

Comment:
This won't work on PowerShell which some users might prefer over CMD... Wondering if there would be a command which works in both terminals 🤔

Reply:
Lets avoid the use of `%userprofile%` alltogether. There isnt really a reason to start "recommending/suggesting" an absolute path based on OS right? For linux/mac we're suggesting a relative path so we probably should for windows too.

Reply:
Is WSL default? I’m thinking about the less technical individual. If it isn’t default, then I would hope that if installed the reader will know to use the Linux instructions

Reply:
> Maybe we instruct them to open WSL before running the same command? That'll also remove the need to have a mac vs windows thing

I dont think this is a viable option as WSL is not enabled by default in windows (very much a developer only feature at the moment)

Reply:
> The default location when you open the terminal in macOS is the user folder.

The same holds true on Windows.

Reply:
Oh. lol. Okay, this is unnecessarily convoluted and I can change it then